WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -96,3 +96,16 @@ class GutenbergSettings {
         }
     }
 }
+
+@objc(GutenbergSettings)
+class GutenbergSettingsBridge: NSObject {
+    @objc(setGutenbergEnabled:forBlog:)
+    static func setGutenbergEnabled(_ isEnabled: Bool, for blog: Blog) {
+        GutenbergSettings().setGutenbergEnabled(isEnabled, for: blog)
+    }
+
+    @objc(postSettingsToRemoteForBlog:)
+    static func postSettingsToRemote(for blog: Blog) {
+        GutenbergSettings().postSettingsToRemote(for: blog)
```

Comment:
Is there any case where we want to set the value locally and not save to the server? Do we need two separate methods or should `setGutenbergEnabled` automatically post to remote?

Reply:
> should setGutenbergEnabled automatically post to remote?

This was the first implementation. I decided to keep them separated since they are two different tasks (local storage/remote sync).

It's also helpful for unit tests

> Is there any case where we want to set the value locally and not save to the server?

Technically yes for self-hosted non-jetpack sites, but that's being handled internally anyway.

Reply:
They are two different tasks, but I don't think the UI that calls this should care. Now that we have both persistence and remote sync, `GutenbergSettings` is starting to act more as a Service class, and I wonder if we should fold all the logic into `EditorSettingsService`, although that's more of a question for the next PR 😁